libqfieldsync/offline_converter.py

```diff
@@ -149,21 +151,30 @@ def _convert(self, project: QgsProject) -> None:
         on_original_project_read = self._on_original_project_read_wrapper(
             xml_elements_to_preserve
         )
-        project.readProject.connect(on_original_project_read)
 
+        temporary_project = QgsProject()
```

Comment:
Are we now allowerd to have instances of `QgsProject` that are not the signleton instance? I remember we had problems with this in the past?

Reply:
@suricactus , in this instance we 100% should.  And the reason why we don't use the QgsProject.instance() singleton is to unlock the massive speed boost by using the QgsProject.FlagDontResolveLayers when read()ing the project. 

If we'd be using that in our main QgsProject.instance(), we would subsequently not be able to manipulate layers.